modules/dynamic-plugins/proc-export-third-party-plugins-rhdh.adoc

```diff
@@ -0,0 +1,124 @@
+[id="proc-export-third-party-plugins-rhdh_{context}"]
+= Exporting third-party plugins in {product}
+
+To use plugins in {product}, the plugins must be exported as derived dynamic plugin package. These packages contain the plugin code and dependencies, ready for dynamic plugin integration into {product-short}.
+
+Backend plugins::
```

Comment:
Consider splitting in multiple procedures:

* _Exporting a third-party backend plugin_

* _Exporting a third-party front-end plugin_

Reply:
The procedure for exporting a backend or frontend plugin is the same; the only difference is the dependencies, which I’ve provided additional details about in separate sections. Therefore, I’ll leave it as is.